client/my-sites/guided-transfer/host-credentials-page/index.jsx

```diff
@@ -0,0 +1,67 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes, Component } from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Bluehost from './bluehost';
+import SiteGround from './siteground';
+import SectionHeader from 'components/section-header';
+
+class HostCredentialsPage extends Component {
+	constructor() {
+		super();
+		this.state = { fieldValues: {} };
+	}
+
+	setFieldValue = ( fieldName, fieldValue ) => {
+		this.setState( { fieldValues: {
+			...this.state.fieldValues,
+			[ fieldName ]: fieldValue,
+		} } );
+	}
+
+	onFieldChange = fieldName => e => {
+		this.setFieldValue( fieldName, e.target.value );
+	}
+
+	submit = data => {
+		// Future PR: This function will trigger the Redux API call
```

Comment:
If we start using a redux store instead of state maybe we can just dispatch `clear field` actions for all of the fields when the `submit` action is dispatched.